samples/dnn/openpose.py

```diff
@@ -57,12 +59,15 @@
 
     frameWidth = frame.shape[1]
     frameHeight = frame.shape[0]
-    inp = cv.dnn.blobFromImage(frame, 1.0 / 255, (inWidth, inHeight),
+    inp = cv.dnn.blobFromImage(frame, inScale, (inWidth, inHeight),
                               (0, 0, 0), swapRB=False, crop=False)
     net.setInput(inp)
     out = net.forward()
 
-    assert(len(BODY_PARTS) == out.shape[1])
+    shape_1 = out.shape[1]
+    if shape_1 == 57: shape_1 = 19
+    if shape_1 == 44: shape_1 = 16
+    assert(len(BODY_PARTS) == shape_1)
```

Comment:
`shape_1` is used only here. We check `assert(len(BODY_PARTS) == out.shape[1])` only to prevent out-of-bound slicing. Can we keep just `assert(len(BODY_PARTS) <= out.shape[1])`?

May I ask you also adapt python sample for HAND model?